spring-integration-file/src/main/java/org/springframework/integration/file/filters/ChainFileListFilter.java

```diff
@@ -31,11 +32,25 @@
  *
  * @author Artem Bilan
  * @author Gary Russell
+ * @author Cengis KOCAURLU
  *
  * @since 4.3.7
  *
  */
 public class ChainFileListFilter<F> extends CompositeFileListFilter<F> {
+	/**
+	* {@inheritDoc}
```

Comment:
I think we can leave without these Javadocs or let's add some formal message as "The constructor matching the super class."

Reply:
i removed this unnecessary javadoc